### ISSUE TITLE
[Bug]: Accessibility issue detected by FastPass on Split Button

### ISSUE THREAD

**Issue by neperezb** [microsoft/fluentui#25847]
### Library

React / v8 (@fluentui/react)

### System Info

```shell
System:
    OS: Windows 10 10.0.19045
    CPU: (8) x64 Intel(R) Xeon(R) CPU E5-1620 v4 @ 3.50GHz
    Memory: 37.56 GB / 63.92 GB
  Browsers:
    Edge: Spartan (44.19041.1266.0), Chromium (107.0.1418.35)
    Internet Explorer: 11.0.19041.1566
```

### Are you reporting Accessibility issue?

yes

### Reproduction

https://developer.microsoft.com/en-us/fluentui#/controls/web/button

## Actual Behavior
Accessibility Insights for Web FastPass flags a nested-interactive issue with the Split Button control in the official documentation page that could cause issues with some screen readers or assistive technologies. The same issue is detected when the component is used/consumed from PowerApps Portal.

![image](https://user-images.githubusercontent.com/25357237/204891349-7c31046b-f6e5-4478-9aec-2bd4039738b2.png)

![image](https://user-images.githubusercontent.com/25357237/204891677-ba95b82f-5983-472a-8c10-2b94a3dc7c3e.png)

## Expected Behavior
Accessibility issue should be addressed if necessary.

### Requested priority

Normal

### Products/sites affected

PowerApps Portal

### Are you willing to submit a PR to fix?

no

### Validations

- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] The provided reproduction is a minimal reproducible example of the bug.

**Comment by khmakoto** [comment]
Hi! This is actually by design, the issue is that we have a control that is a single keyboard tab/arrow stop, but has two actions (the primary button and opening a menu). We can't remove either the outer interactive role or the inner ones without degrading practical accessibility, and we can't switch to a two-button and two-tab-stop approach without introducing a breaking change.

We've done work to mitigate the practical accessibility issues with this approach, so while it may be quirky with various screen readers, it shouldn't violate WCAG or be blocking to anyone. More context here: https://github.com/microsoft/fluentui/issues/18484#issuecomment-861723118

The v9 `SplitButton` does use two separate buttons with separate tab/arrow stops 😊
